MonkeyWrench.Web.WebService/Scheduler/Scheduler.cs

```diff
@@ -120,7 +120,7 @@ public static bool ExecuteScheduler (bool forcefullupdate)
 				reports = GetReports (forcefullupdate);
 
 				using (DB db = new DB (true)) {
-					lanes = db.GetAllLanes ();
+					lanes = db.GetAllLanes ().FindAll(lane => lane.enabled);

```

Comment:
We're already filtering disabled lanes a few lines below, so this doesn't improve much (it'll prevent a line from being written to the log, and that's about it).